d2l-table-style.html

```diff
@@ -4,7 +4,7 @@
 <dom-module id="d2l-table-style">
 	<template>
 		<style include="d2l-colors">
-			:host .table-container > ::content > table {
+			:host .table-container > ::content table[is="d2l-table"] {
```

Comment:
Yes. I couldn't use .wrapper because it is inside the d2l-scroll-wrapper shadow dom.